frontend/src/container/LogsExplorerChart/index.tsx

```diff
@@ -2,25 +2,29 @@ import Graph from 'components/Graph';
 import Spinner from 'components/Spinner';
 import getChartData, { GetChartDataProps } from 'lib/getChartData';
 import { colors } from 'lib/getRandomColor';
-import { memo, useMemo } from 'react';
+import { memo, useCallback, useMemo } from 'react';
 
 import { LogsExplorerChartProps } from './LogsExplorerChart.interfaces';
 import { CardStyled } from './LogsExplorerChart.styled';
 
 function LogsExplorerChart({
 	data,
 	isLoading,
+	isLabelEnabled = true,
 }: LogsExplorerChartProps): JSX.Element {
-	const handleCreateDatasets: Required<GetChartDataProps>['createDataset'] = (
-		element,
-		index,
-		allLabels,
-	) => ({
-		label: allLabels[index],
-		data: element,
-		backgroundColor: colors[index % colors.length] || 'red',
-		borderColor: colors[index % colors.length] || 'red',
-	});
+	const handleCreateDatasets: Required<GetChartDataProps>['createDataset'] = useCallback(
+		(element, index, allLabels) => ({
+			data: element,
+			backgroundColor: colors[index % colors.length] || 'red',
```

Comment:
In place of `red`, we should be using the hex from our figma color 

@kaushikmacharla07  :  Please help with the red color hex we want to use across the product.